rotkehlchen/chain/evm/decoding/curve/lend/balances.py

```diff
@@ -29,12 +30,15 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
-class CurveLendBalances(ProtocolWithBalance):
+class CurveControllerCommonBalances(ProtocolWithBalance, ABC):
+
     def __init__(
             self,
             evm_inquirer: 'EvmNodeInquirer',
             tx_decoder: 'EVMTransactionDecoder',
+            controller_extra_data_key: Literal['vault_controller', 'crvusd_controller'],
```

Comment:
same here. As in the other place. Why add this extra logic just for the name of a dict key? Just find a common name to use that describes both. Perhaps `controller_address`?